app/src/utils/record-cache/index.ts

```diff
@@ -12,10 +12,18 @@ export enum IappRecordMode {
   Record = 'record',
   Row = 'row'
 }
+
+export enum CacheDownloadMode {
+  DEFAULT = '',
+  PAUSE = 'pause',
+  ABORT = 'abort'
```

Comment:
```suggestion
  DEFAULT,
  PAUSE,
  ABORT
```

As an FYI, not assigning values just auto-assigns them `0, 1, 2...` 
I generally just use strings if I want to use the values for something in the UI.

Reply:
Good to know! 
I like the one that's in place now, it's more descriptive and understandable.